packages/typescript/src/browser/typescript-frontend-contribution.ts

```diff
@@ -83,11 +79,6 @@ export class TypeScriptFrontendContribution implements CommandContribution, Menu
             isEnabled: () => !!this.clientContribution.logFileUri,
             isVisible: () => !!this.clientContribution.logFileUri
         });
-        commands.registerCommand(TypeScriptCommands.restartServer, {
-            execute: () => this.clientContribution.restart(),
-            isEnabled: () => this.clientContribution.running,
```

Comment:
We could implement `isEnabled` and `isVisible` in the same way for generic restart command that it is not visible while a server is restarting.

Reply:
(y)